src/codec_svt.c

```diff
@@ -161,6 +161,17 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
             svt_config->profile = HIGH_PROFILE;
         }
 
+        // In order for SVT-AV1 to force keyframes by setting pic_type to
+        // EB_AV1_KEY_PICTURE on any frame, force_key_frames has to be set.
+        svt_config->force_key_frames = 1;
+
+        // keyframeInterval == 1 case is handled when encoding each frame by
+        // settings pic_type to EB_AV1_KEY_PICTURE. For keyframeInterval > 1,
```

Comment:
settings => setting

Note: It would be good to mention that setting intra_period_length to 0 doesn't work, even though it should.

I found the following comment and code in SVT-AV1:

```
            // If the Intra period length is 0, then introduce an intra for every picture
            if (scs->static_config.intra_period_length == 0)
                pcs->cra_flag = TRUE;
```

Reply:
Done.